go.mod

```diff
@@ -2,185 +2,110 @@ module github.com/signalfx/signalfx-agent
 
 go 1.14
 
-replace git.apache.org/thrift.git => github.com/apache/thrift v0.0.0-20180902110319-2566ecd5d999
-
-replace code.cloudfoundry.org/go-loggregator => github.com/signalfx/go-loggregator v1.0.1-0.20200205155641-5ba5ca92118d
-
-replace github.com/influxdata/telegraf => github.com/signalfx/telegraf v0.10.2-0.20201211214327-200738592ced
-
-replace github.com/dancannon/gorethink => gopkg.in/gorethink/gorethink.v4 v4.0.0
-
-replace github.com/signalfx/signalfx-agent/pkg/apm => ./pkg/apm
+replace (
+	code.cloudfoundry.org/go-loggregator => github.com/signalfx/go-loggregator v1.0.1-0.20200205155641-5ba5ca92118d
+	github.com/dancannon/gorethink => gopkg.in/gorethink/gorethink.v4 v4.0.0
+	github.com/influxdata/telegraf => github.com/signalfx/telegraf v0.10.2-0.20201211214327-200738592ced
+	github.com/signalfx/signalfx-agent/pkg/apm => ./pkg/apm
+	github.com/soheilhy/cmux => github.com/rmfitzpatrick/cmux v0.1.4 // required to drop google.golang.org/grpc/examples/helloworld/helloworld test dep
```

Comment:
Take this repo and put it into a new directory in this repo called `thirdparty/cmux` and make the replace reference that dir instead of your personal fork.  Would rather not rely on personal forks.

Reply:
What about forking in the `signalfx` project to avoid taking on vendored indirect deps?

Reply:
That would be better than a personal fork but putting it here in the repo avoids cluttering our github org with random packages.  It looks like a fairly small number of files so wouldn't be a huge burden here.  Whichever you prefer.

Reply:
Updated with your initial request since go modules will let other libraries' replace directives use the new path and a standalone repo isn't necessary.